cmd/docker-app/deploy.go

```diff
@@ -51,10 +55,33 @@ func deployCmd(dockerCli command.Cli) *cobra.Command {
 	cmd.Flags().BoolVarP(&opts.deploySendRegistryAuth, "with-registry-auth", "", false, "Sends registry auth")
 	if internal.Experimental == "on" {
 		cmd.Flags().StringArrayVarP(&opts.deployComposeFiles, "compose-files", "c", []string{}, "Override Compose files")
+		cmd.Flags().BoolVarP(&opts.deployNoRenderConfig, "disable-render-config", "", false, "Do not apply renderer to configuration files")
 	}
 	return cmd
 }
 
+func renderAttachments(app *types.App, tempDir string, d map[string]string) error {
+	for _, a := range app.Attachments() {
+		content, err := ioutil.ReadFile(filepath.Join(app.Path, a.Path()))
```

Comment:
What if the file is big, like say 3 GB...? I always wonder what is the best practice in that case... Stat the file before and skip if it's "too" big? (but then we will have to define what is too big).